tcf_website/templates/reviews/zero_hours_modal.html

```diff
@@ -0,0 +1,21 @@
+<!-- User Submitted Zero Hours in Review -->
+<div class="modal fade" id="zero-hours-review" tabindex="-1" role="dialog" aria-labelledby="zero-hours-label" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Zero Hours per Week in Review</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+          It looks like you submitted zero total hours of work per week for this class. Please double check
```

Comment:
Minor grammar nitpicks (lmao), feel free to override: I _think_ `double-check` is a hyphenated compound word - could be wrong on that one though. Also I'm pretty sure there should be a comma between `work` and `as`?

Reply:
From what I can tell either is fine I think? I'll change it to double-check to be safe though. Should be a comma between "work" and "as" like you mentioned though